frontend/src/metabase/lib/dom.js

```diff
@@ -495,23 +495,10 @@ export function initializeIframeResizer(onReady = () => {}) {
       onReady,
     };
 
-    // FIXME: Crimes
-    // This is needed so the FE test framework which runs in node
-    // without the avaliability of require.ensure skips over this part
-    // which is for external purposes only.
-    //
-    // Ideally that should happen in the test config, but it doesn't
-    // seem to want to play nice when messing with require
-    if (typeof require.ensure !== "function") {
-      return false;
-    }
-
     // Make iframe-resizer avaliable to the embed
     // We only care about contentWindow so require that minified file
 
-    require.ensure([], require => {
-      require("iframe-resizer/js/iframeResizer.contentWindow.js");
-    });
+    import("iframe-resizer/js/iframeResizer.contentWindow.js");
```

Comment:
This has been broken since the SWC change https://github.com/metabase/metabase/commit/e02589a386814a83aaf330db61d4377e129eeb95

Reply:
wow, why?

Reply:
IDK, but it just didn't import `iframe-resizer/js/iframeResizer.contentWindow.js` with `require.ensure`. I'd been troubleshooting this for so long until I tried using your code `import()` and it imported this module correctly 🤷. From the behavior, it seems this syntax doesn't work with SWC.

Reply:
oh, you're right, swc is the reason

https://github.com/swc-project/swc/issues/7505
https://github.com/webpack/webpack/discussions/17849

that's great we didn't backport this change and you noticed it 😄